Examples/StockQuoteConnector_final.html

```diff
@@ -31,26 +34,63 @@
     return num < 9 ? "0" + num.toString() : num.toString();
   }
 
+  function getTickersArrayFromConnectionData(connectionData) {
+    var tickersArray = connectionData.split(',');
+    
+    // Remove whitespace from each ticker string
+    return tickersArray.map(function(obj) {
+      return obj.trim(); 
+    });
+  }
+
   //
   // Connector definition
   // 
-
   var myConnector = tableau.makeConnector();
 
-  myConnector.getColumnHeaders = function() {
-    var fieldNames = ['Ticker', 'Day', 'Close'];
-    var fieldTypes = ['string', 'date', 'float'];
-    tableau.headersCallback(fieldNames, fieldTypes); // tell tableau about the fields and their types
+  myConnector.getSchema = function(schemaCallback) {
+    var tickersArray = getTickersArrayFromConnectionData(tableau.connectionData);
+    
+    var schema = [];
+    _.forEach(tickersArray, function(ticker) {
+      var col1 = { id: "Ticker", dataType: "string"};
+      var col2 = { id: "Day", dataType: "date"};
+      var col3 = { id: "Close", dataType: "float"};
+      var cols = [col1, col2, col3];
+      
+      var tableInfo = { 
+        id: ticker,
+        columns: cols
+      };
+      
+      schema.push(tableInfo);
+    });
+   
+    schemaCallback(schema); // tell tableau about the fields and their types
   };
    
-  myConnector.getTableData = function(lastRecordToken) {
+  myConnector.getData = function(tables, doneCallback) {
```

Comment:
Don't feel particularly strongly on any of the above--I haven't see anything fundamentally limiting in these changes that couldn't be handled in user-land.